doc/api/vm.md

```diff
@@ -286,33 +351,21 @@ console.log('localVar: ', localVar);
 // evalResult: 'eval', localVar: 'eval'
 ```
 
-`vm.runInThisContext()` does not have access to the local scope, so `localVar`
-is unchanged. [`eval()`][] does have access to the local scope, so `localVar` is
-changed.
-
-In this way `vm.runInThisContext()` is much like an [indirect `eval()` call][],
-e.g. `(0,eval)('code')`. However, it also has the following additional options:
-
-- `filename`: allows you to control the filename that shows up in any stack
-  traces produced.
-- `lineOffset`: allows you to add an offset to the line number that is
-  displayed in stack traces
-- `columnOffset`: allows you to add an offset to the column number that is
-  displayed in stack traces
-- `displayErrors`: if `true`, on error, attach the line of code that caused
-  the error to the stack trace. Will capture both syntax errors from compiling
-  `code` and runtime errors thrown by executing the compiled code. Defaults to
-  `true`.
-- `timeout`: a number of milliseconds to execute `code` before terminating
-  execution. If execution is terminated, an [`Error`][] will be thrown.
-
-## Example: Run a Server within a VM
-
-The context of `.runInThisContext()` refers to the V8 context. The code passed
-to this VM context will have it's own isolated scope. To run a simple web server
-using the `http` module, for instance, the code passed to the context must either
-call `require('http')` on its own, or have a reference to the `http` module passed
-to it. For instance:
+Because `vm.runInThisContext()` does not have access to the local scope,
+`localVar` is unchanged. In contrast, [`eval()`][] *does* have access to the
+local scope, so the value `localVar` is changed. In this way
+`vm.runInThisContext()` is much like an [indirect `eval()` call][], e.g. 
+`(0,eval)('code')`.
+
+## Example: Running an HTTP Server within a VM
+
+When using either `script.runInThisContext()` or `vm.runInThisContext()`, the
+code is executed within the current V8 global context. The code passed
+to this VM context will have it's own isolated scope.
```

Comment:
grr... my dang muscle memory on typing that stupid ' every time I want to type `its` it's freaking annoying...